Tribler/Core/Modules/watch_folder.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import os
+from twisted.internet.task import LoopingCall
+from Tribler.Core.TorrentDef import TorrentDef
+from Tribler.Core.Utilities.utilities import fix_torrent
+from Tribler.dispersy.taskmanager import TaskManager
+
+
+WATCH_FOLDER_CHECK_INTERVAL = 120
```

Comment:
Isn't two minutes a lot? It's not like this is an expensive operation. Maybe make it 10 secs?